### PR TITLE
create README.md and initialize yarn

### LINKED ISSUE #1: npm not Initialized
## Issue

The project is missing package.json file, and it is needed to be created for installing all the dependencies with ease.

## Would you like to work on this issue?

Yes

@iiitv/project-maintainers

### PR DESCRIPTION
**Pull request by Shreyansh-Mishra**
this commit fixes #1   
@iiitv/project-maintainers

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "express-session": "^1.17.1",
+    "express-validator": "^6.6.1",
+    "mongoose": "^5.10.2",
+    "passport": "^0.4.1",
+    "passport-local": "^1.0.0",
+    "passport-local-mongoose": "^6.0.1",
+    "yarn.lock": "^0.0.1-security"
```
Comment (anshumanv) [inline package.json]: this seems wrong

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+```bash
+git clone https://github.com/iiitv/Club-Collabaration-backend/
+cd Club-Collabaration-backend
+yarn install
+```
+## Running
+
+```bash
+node app.js
```
Comment (anshumanv) [inline README.md]: add a running script in package.json and give start command as `yarn start` or something

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "club-collabaration-backend",
+  "version": "1.0.0",
+  "main": "index.js",
```
Comment (anshumanv) [inline package.json]: there's no main, you can delete this

File: package.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "club-collabaration-backend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/iiitv/Club-Collabaration-backend",
+  "author": "",
```
Comment (anshumanv) [inline package.json]: remove this too

**Review by anshumanv — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Shreyansh-Mishra) [reply package.json]: It is correct, all of them are independent dependencies.

File: README.md (hunk 2 above)
Comment (Shreyansh-Mishra) [reply README.md]: Added running script in package.json, and updated it in README.md

File: package.json (hunk 3 above)
Comment (Shreyansh-Mishra) [reply package.json]: Removed unnecessary fields.

File: package.json (hunk 4 above)
Comment (Shreyansh-Mishra) [reply package.json]: Removed unnecessary fields.

File: package.json (hunk 1 above)
Comment (anshumanv) [reply package.json]: Which file uses `yarn.lock` dependency?

**Review by anshumanv — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Shreyansh-Mishra) [reply package.json]: removed ``yarn.lock`` dependency, it might have been added accidentally.

**Comment by anshumanv** [comment]
Thanks!

**Pull request merged**